mon-pix/app/templates/components/qcm-proposals.hbs

```diff
@@ -6,7 +6,7 @@
          id="checkbox_{{inc index}}"
          checked={{labeledCheckbox.[1]}}
```

Comment:
[D'après la doc](https://guides.emberjs.com/release/components/built-in-components/#toc_checkboxes), il faut mettre `@type` et `@checked`